src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy

```diff
@@ -49,42 +53,43 @@ final class ACESeqMethods {
                 ACEseqConstants.TOOL_IMPUTE_GENOTYPES,
                 controlBam,
                 null,
-                ACEseqConstants.PARM_CHR_INDEX,
-                getGlobalJobSpecificParameters(controlBam.executionContext.configuration))
+                ACEseqConstants.PARM_CHR_INDEX)
+//        ,
+//                getGlobalJobSpecificParameters(controlBam.executionContext.configuration))
```

Comment:
dito

Reply:
Also dito! I won't merge this without a working version.